Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.yml

```diff
@@ -0,0 +1,196 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: NucleonCyberFeed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+# - defaultvalue: https://openphish.com/feed.txt
+#   display: Server's URL
+#   additionalinfo: Should be feed of type .txt. For using other types of feed, modify
+#     the parsing in the python file.
+#   name: url
+#   required: true
+#   type: 0
```

Comment:
You are using url parameter in the integration, but you have commented it out here. Any reason for that?